spec_7/index.js

```diff
@@ -0,0 +1,36 @@
+import { identifiers } from './resources'
+import converter from './dateConverter.js'
+
+const dateFile = 'spec_7/dates.txt'
+const dates = converter.getRawDates( dateFile )
+const validFormats = converter.getValidFormats( dates )
+
+const invalidFormats = {
+  slashies: converter.filterFormats( dates, identifiers.SLASH ),
```

Comment:
Consider using identifier names that would be easy to understand for *anyone* reading your code - since English is my native language, I can easily inuit the intent of these identifiers, but someone who did not speak English as their first language might struggle to understand the meaning of these identifiers.